Microsoft.Toolkit.Uwp/Helpers/BluetoothLEHelper/GattConvert.cs

```diff
@@ -0,0 +1,148 @@
+﻿// ******************************************************************
+// Copyright (c) Microsoft. All rights reserved.
+// This code is licensed under the MIT License (MIT).
+// THE CODE IS PROVIDED “AS IS”, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+// INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+// IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+// DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+// TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH
+// THE CODE OR THE USE OR OTHER DEALINGS IN THE CODE.
+// ******************************************************************
+
+using System;
+using System.Text;
+using Windows.Security.Cryptography;
+using Windows.Storage.Streams;
+
+namespace Microsoft.Toolkit.Uwp
+{
+    /// <summary>
+    /// Extension methods for Gatt Convert.
+    /// </summary>
+    public static class GattConvert
+    {
+        /// <summary>
+        /// Convert the string to an IBuffer
+        /// </summary>
+        /// <param name="data">The data.</param>
+        /// <returns>An IBuffer of data.</returns>
+        public static IBuffer ToIBuffer(string data)
+        {
+            var writer = new DataWriter();
+            writer.WriteString(data);
+
+            return writer.DetachBuffer();
+        }
+
+        /// <summary>
+        /// Convert a hex to an IBuffer
+        /// </summary>
+        /// <param name="hex">A hex value.</param>
+        /// <returns>An IBuffer</returns>
+        public static IBuffer ToIBufferFromHexString(string hex)
+        {
+            hex = hex.Replace("-", string.Empty);
+
+            var numberChars = hex.Length;
+            var bytes = new byte[numberChars / 2];
+
+            for (var i = 0; i < numberChars; i += 2)
+            {
+                bytes[i / 2] = Convert.ToByte(hex.Substring(i, 2), 16);
+            }
+
+            var writer = new DataWriter();
```

Comment:
It's this a possible leak? Why are these 3 methods (ToIBuffer, ToIBufferFromHexString, ToIBuffer) added if they are not being used? Should the toolkit expose them?